governance/votes/Executive Vote - September 27, 2024.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Template - [Executive Vote] SBE Updates, Sky Ecosystem Liquidity Bootstrapping, Aligned Delegate Compensation, and Spark Proxy Spell - September 27, 2024
+summary: Smart Burn Engine Parameter Updates, USDS and SKY Transfers for Sky Ecosystem Liquidity Bootrapping, August MKR and DAI Aligned Delegate Compensation, and trigger Spark Proxy Spell.
```

Comment:
The SBE changes are more than parameter updates. It is an entirely new contract deployment that is changing from buy-and-make to buy-and-burn